cubedash/_stac.py

```diff
@@ -203,8 +203,9 @@ def as_stac_item(dataset: DatasetItem) -> pystac.Item:
             dataset_doc.lineage
         ) == 1:
             # From old to new lineage type.
-            dataset_doc.lineage = {
-                classifier: [dataset["id"]]
+            # FIXME: remove type ignores and fix the issues.
+            dataset_doc.lineage = {  # type: ignore[misc]
+                classifier: [dataset["id"]]  # type: ignore[has-type]
```

Comment:
I'm pretty sure this code is wrong, but I don't know how to fix it. The code has been there for several years already so just add ignores and a FIXME.